### ISSUE TITLE
Why the predict_proba() doesn't work

### ISSUE THREAD

**Issue by xiaoxiaoyaozi** [keras-team/keras#10607]
I have a x_train with 34 dimension, which are some stock technical indicator data. And y_train is label, which is binary(1,0) represent the stock is uptrend or downtrend. x_test is the stock technical indicator data thich I want to use to predict trend probability. Here is my code.
```
model = Sequential()
model.add(Dense(512, activation='relu', input_dim=34))
model.add(Dense(200, activation='relu'))
model.add(Dense(200, activation='relu'))
model.add(Dense(128, activation='relu'))
model.add(Dense(1, activation='softmax'))
model.compile(optimizer='rmsprop',
              loss='binary_crossentropy',
              metrics=['binary_accuracy'])

model.fit(x_train, y_train, epochs=2, batch_size=50, verbose=0)
print(model.predict_proba(x_test, batch_size=50))
```
However I got this:
```
[[0.]
 [0.]
 [0.]
 ...
 [0.]
 [0.]
 [0.]]
```
How can I get a pair of proba about class '1' and class '0'? Like this:
```
[  4.93983855e-04,   2.28181913e-01],
 [  3.14530940e-03,   9.60477769e-01]
```

**Comment by abhaikollara** [comment]
I think the correct activation to use for the final layer in binary classification is `sigmoid`. That will give you a value between 0 and 1 for `predict_proba`
